protocol_test.go

```diff
@@ -87,7 +87,7 @@ func TestLongCall(t *testing.T) {
 
 	longResult, err := cli.LongCall(context.Background())
 	require.Nil(t, err, "call should have succeeded")
-	require.Equal(t, longResult, 100, "call should have succeeded")
+	require.Equal(t, longResult, 10, "call should have succeeded")
```

Comment:
Is this regarding the race you mentioned encountering? Would it still happen after the above changes re. the goroutine?